app/client/components/footer/footer.tsx

```diff
@@ -224,15 +232,25 @@ export const Footer = () => {
               <div css={footerMenuStyles}>
                 {footerLinks.map((linkList, i) => (
                   <ul key={i} css={footerMenuUlStyles}>
-                    {linkList.map(({ title, link, onClick, USAonly }) => {
-                      return USAonly && !isInUSA ? null : (
-                        <li key={title} css={footerMenuLiStyles}>
-                          <a
-                            href={link}
-                            onClick={onClick}
-                            css={footerLinkStyles}
-                          >
-                            {title}
+                    {linkList.map(({ title, titleUSA, link, cmp }) => {
+                      const actualTitle = cmp && isInUSA ? titleUSA : title;
+                      const extraParams = {
+                        ...(cmp
+                          ? {
+                              href: "#",
+                              onClick: () => {
+                                if (importedCmp) {
+                                  importedCmp.showPrivacyManager();
+                                }
```

Comment:
not a biggy, but could use optional chaining and avoid the if:

`importedCmp?.showPrivacyManager();`

Reply:
Done!